zgw/zaken-api/src/main/kotlin/nl/nlportal/zakenapi/client/request/ZakenImpl.kt

```diff
@@ -48,11 +48,20 @@ class SearchZakenImpl(val zakenApiClient: ZakenApiClient) : SearchZaken {
         return this
     }
 
+    override fun isOpen(open: Boolean): SearchZaken {
+        queryParams.add("einddatum__isnull", open.toString())
+        return this
+    }
+
     override fun ofZaakType(zaakType: String): SearchZaken {
         queryParams.add("zaaktype", zaakType)
         return this
     }
 
+    override fun ofZaakTypes(zaakTypes: List<String>): SearchZaken {
+        return this
```

Comment:
Ik denk dat hier nog wat code mist? of heeft de normale zaken endpoint geen list of zaaktypes?
dan misschien beter een error gooien hier.